client/components/app.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import axios from 'axios';
+import PartySize from './partySize.jsx';
+import Date from './date.jsx';
+import Time from './time.jsx';
+
+class App extends React.Component { 
+	constructor(props) {
+		super(props);
+		this.state = {
+			restaurantId: '',
+		}
+	} 
+	getInfo () {
+		axios.get('/restaurant/:restaurant_id')
+			.then(function(reponse) {
+				console.log(response);
+				this.setState({
+					restaurantId: response
+				})
+			}); 	
+	}
+	render () {
+		return (
+			<div id="appBox">
```

Comment:
This is really generic and maybe something that doesn't need to be addressed. But I've heard that using id attributes in HTML elements should generally be used sparingly just because it's so selective. Classes are usually preferred whenever id attributes aren't required. Just at first glance, it seems like there's a lot of id attributes on this section of regex. Potentially refactor any that don't need to be ids?